common/src/chain/config/mod.rs

```diff
@@ -188,9 +208,13 @@ const MAX_BLOCK_HEADER_SIZE: usize = 1024;
 const MAX_BLOCK_TXS_SIZE: usize = 524_288;
 const MAX_BLOCK_CONTRACTS_SIZE: usize = 524_288;
 const TOKEN_MIN_ISSUANCE_FEE: Amount = Amount::from_atoms(10_000_000_000_000);
-const TOKEN_MAX_URI_LEN: usize = 1024;
 const TOKEN_MAX_DEC_COUNT: u8 = 18;
 const TOKEN_MAX_TICKER_LEN: usize = 5;
+const TOKEN_MIN_HASH_LEN: usize = 32;
+const TOKEN_MAX_HASH_LEN: usize = 4;
```

Comment:
min is 32 and max is 4?

Reply:
How is this not triggering tests failures? Please investigate how this is not breaking everything.

Reply:
As I mentioned in DM, this problem was on my side. After updating rust I didn't call cargo clean and for some reasons tests done fine even if the code changed. I fixed it and added a new test for NFT hash.